components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2680,10 +2680,10 @@ def su(request, user, conn=None, **kwargs):
         return HttpResponse(t.render(c))
 
 
-def _annotations(request, objtype, objid, conn=None, **kwargs):
+def _bulk_file_annotations(request, objtype, objid, conn=None, **kwargs):
```

Comment:
Perhaps:

```
def _annotations(...):
    warning....
    return self._bulk_file_annotations(...)

def _bulk_file_annotations(...):
    ....
```

I know this is prefixed with an underscore, but the limited cost of adding 4-5 lines to avoid the possibility of any user pain in a patch release seems worth it.

Reply:
Hah, I was just coming to this PR to say that OMERO.parade is actually broken with this change on web-dev-merge.